src/ultils/component.js

```diff
@@ -0,0 +1,32 @@
+import displayListItems from './template.js';
+
+const addItem = (items, newTodo) => {
+  items.push({
+    index: items.length + 1,
+    description: newTodo,
+    completed: false,
+  });
+  localStorage.setItem('todoList', JSON.stringify(items));
+};
+
+const init = (data) => {
+  data = localStorage.getItem('todoList');
+  let getData = data;
+  if (getData === null) {
+    getData = [];
+  } else {
+    getData = JSON.parse(getData);
+  }
+  return getData;
+};
```

Comment:
what about if you made this function in a separate module and use it whenever you want instead of repeating this lines every time you need to bring data from your local storage